src/middlewares/error.middleware.js

```diff
@@ -0,0 +1,13 @@
+import { ApiError } from '../exceptions/api-error';
+
+export const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    const { status, message, errors } = error;
+
+    res.status(status).send({ message, errors });
+  }
+
+  res.status(500).send({
```

Comment:
The error middleware does not call `next()` after handling an `ApiError`. This means that if an `ApiError` is caught, the middleware will not pass control to the next middleware in the stack. Consider adding `return` before `res.status(status).send({ message, errors });` to prevent the execution of the next `res.status(500).send()` block.